src/GuessHistory.tsx

```diff
@@ -33,7 +42,26 @@ export default function GuessHistory() {
     
     return (
         <Flex marginLeft="auto" marginRight="auto">
-            {getPosterButtons()}
+            {guessList.map(guess => (
+                <Button
+                backgroundImage={`url(${guess.poster})`}
+                backgroundPosition="center"
+                backgroundSize="cover"
+                _hover={{}}
+                _active={{}}
```

Comment:
can remove these attributes if we're not using it

Reply:
Hover and active are being used to git rid of the default grey hover and active background colors.